react/features/device-selection/functions.js

```diff
@@ -29,3 +42,143 @@ export function getDeviceSelectionDialogProps(stateful: Object | Function) {
         selectedVideoInputId: settings.cameraDeviceId
     };
 }
+
+/**
+ * Processes device requests from external applications.
+ *
+ * @param {Dispatch} dispatch - The redux {@code dispatch} function.
+ * @param {Function} getState - The redux function that gets/retrieves the redux
+ * state.
+ * @param {Object} request - The request to be processed.
+ * @param {Function} responseCallback - The callback that will send the
+ * response.
+ * @returns {boolean}
```

Comment:
What is the boolean for?

Reply:
Added a comment in the JS doc:
`True if the request has been processed and false otherwise.`